ui/src/common/storage.js

```diff
@@ -6,59 +6,36 @@ localforage.config({
   version: 1.0,
 });
 
-const rewriteUserSettingsFromLocalStorage = async () => 
-  Promise.all(Object.entries(localStorage).map(async ([key, value]) => localforage.setItem(key, value)));
+export const rewriteLocalStorage = async () => {
+  await Promise.all(Object.entries(localStorage).map(([key, value]) => {
+    // With localforage we are not limited to strings like in localStorage,
+    // therefore, we can try to re-write all the JSONs to JS objects.
+    try {
+      return localforage.setItem(key, JSON.parse(value));
+    } catch (error) {
+      return localforage.setItem(key, value);
+    }
+  }));
 
-// adapted from https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Testing_for_availability
-async function isStorageAvailable() {
-  let storage;
-  try {
-    // eslint-disable-next-line dot-notation
-    storage = localforage;
-    const test = '__test__';
-    await storage.setItem(test, test);
-    await storage.removeItem(test);
-    await rewriteUserSettingsFromLocalStorage();
-    return true;
-  } catch (e) {
-    return (
-      e instanceof DOMException &&
-      (e.code === 22 ||
-        e.code === 1014 ||
-        e.name === 'QuotaExceededError' ||
-        e.name === 'NS_ERROR_DOM_QUOTA_REACHED') &&
-      (storage && storage.length !== 0)
-    );
-  }
+  localStorage.clear();
```

Comment:
this will delete all of the user's localstorage 🙈

Reply:
Actually that's what I wanted to do - delete everything that is in the inspire domain's localStorage (I checked and it's separated by domain, so it will not mess with users'  other domains info stored in localStorage), because we don't need it anymore, we only use indexedDB now. And if we keep it, next time the user visits the page localStorage will be once again rewritten to LF and the new search history, page size and everything from the last session will be lost and replaced by legacy info from LS.